server/stream.go

```diff
@@ -96,13 +96,19 @@ type PubAck struct {
 
 // StreamInfo shows config and current state for this stream.
 type StreamInfo struct {
-	Config  StreamConfig        `json:"config"`
-	Created time.Time           `json:"created"`
-	State   StreamState         `json:"state"`
-	Domain  string              `json:"domain,omitempty"`
-	Cluster *ClusterInfo        `json:"cluster,omitempty"`
-	Mirror  *StreamSourceInfo   `json:"mirror,omitempty"`
-	Sources []*StreamSourceInfo `json:"sources,omitempty"`
+	Config     StreamConfig        `json:"config"`
+	Created    time.Time           `json:"created"`
+	State      StreamState         `json:"state"`
+	Domain     string              `json:"domain,omitempty"`
+	Cluster    *ClusterInfo        `json:"cluster,omitempty"`
+	Mirror     *StreamSourceInfo   `json:"mirror,omitempty"`
+	Sources    []*StreamSourceInfo `json:"sources,omitempty"`
+	Alternates []StreamAlternate   `json:"alternates,omitempty"`
+}
+
+type StreamAlternate struct {
+	Name    string `json:"name"`
+	Cluster string `json:"cluster"`
```

Comment:
Domain?

Reply:
This is to be able to select a read replica that is closer to you than the source.

Reply:
Right now I don't think we can cross accounts or domains.

Reply:
yes It's possible to source/mirror cross domains

Reply:
ok so you think we should include domain even if the majority of the time it will be the default or the same?

Remember we can always add things, once we do though hard if not impossible to remove them. Hence I like to start small and provide less to start.

Reply:
I added.